docs/data_types/image/image.md

```diff
@@ -0,0 +1,156 @@
+# 🖼️ Image
+
+DocArray supports many different modalities including the widely used `Image` modality.
+This section will show you how to load and handle image data using DocArray.
+
+Moreover, we will introduce DocArray's image specific types, to represent your image data ranging from [`ImageUrl`][docarray.typing.url.ImageUrl] to [`ImageBytes`][docarray.typing.bytes.ImageBytes] and [`ImageNdArray`][docarray.typing.tensor.image.image_ndarray.ImageNdArray].
+
+!!! note
+    This requires `Pillow` dependency. You can install all necessary dependencies via:
+    ```cmd 
+    pip install "docarray[image]"
+    ```
+
+## Load image
+
+First, let's define our class `MyImage`, which extends [`BaseDoc`][docarray.base_doc.doc.BaseDoc] and has an `url` attribute of type [`ImageUrl`][docarray.typing.url.ImageUrl], as well as an optional `tensor` attribute of type `ImageTensor`.
+
+Next, let's instantiate a `MyImage` object with a local or remote url. 
+
+```python
+from docarray.typing import ImageTensor, ImageUrl
+from docarray import BaseDoc
+
+
+class MyImage(BaseDoc):
+    url: ImageUrl
+    tensor: ImageTensor = None
+
+
+img = MyImage(url='apple.png')
+```
+
+To load the image data you can simply call [`.load()`][docarray.typing.url.ImageUrl.load] on the `url` attribute. By default, [`ImageUrl.load()`][docarray.typing.url.ImageUrl.load] returns an [`ImageNdArray`][docarray.typing.tensor.image.image_ndarray.ImageNdArray] object.
```

Comment:
```suggestion
To load the image data you can call [`.load()`][docarray.typing.url.ImageUrl.load] on the `url` attribute. By default, [`ImageUrl.load()`][docarray.typing.url.ImageUrl.load] returns an [`ImageNdArray`][docarray.typing.tensor.image.image_ndarray.ImageNdArray] object.
```
In general it's good to avoid words like "simply" or "easily", because, inevitably, for some users out there it will not be simple or easy. And we don't want them to feel stupid :)